### DISCUSSION TITLE
Zones don't exist when I try to see their forecast

### DISCUSSION THREAD

**Discussion by ocket8888** [weather-gov/api#702] [General]
It seems like there's no Zone that I can find that will give me a forecast. For example: Adams County, Colorado. ZoneID: `CO001`.

- https://api.weather.gov/zones/county/COC001 - 200 OK gives me details about the zone.
- https://api.weather.gov/zones/county/COC001/forecast - 404 Not Found _"Forecast zone COC001 does not exist"_ (???????)
- https://api.weather.gov/zones/forecast/COC001 - 404 Not Found _"Forecast zone COC001 does not exist"_ (?????)

So, uh... that zone clearly exists. Because I can see it, right there in the API. Also I've seen the place with my own eyes, so I know it's no fantasy realm or hilarious prank (and also I've tried three other counties to the same effect). Why doesn't this zone exist anymore when I look at it too closely?

**Comment by akrherz** [comment]
The UGC form for the zone is `COZ001`, your first link fails and is not a 200 OK

**Reply by ocket8888** [reply]
Oh, I forgot a letter. The current content is correct.

**Comment by ktrue** [comment]
It's my understanding that only forecast zones (ssZnnn) have forecasts.  County zones (ssCnnn) don't have forecasts, but may have alerts.

**Reply by ocket8888** [reply]
Checking the response from `/zones`, it doesn't seem that "forecast" is a zone type. I see

- public
- fire
- coastal
- county
- offshore

How do I know from inspecting a zone whether or not the forecast endpoint(s) is/are going know if they exist?

**Reply by JJKraw** [reply]
https://api.weather.gov/zones?type=forecast

These are the zones for which you can issue https://api.weather.gov/zones/forecast/ssZnnn/forecast as far as I know.
